code/modules/mob/living/simple_animal/slime/slime.dm

```diff
@@ -73,10 +81,30 @@
 	var/coretype = /obj/item/slime_extract/grey
 	var/list/slime_mutation[4]
 
-	var/static/list/slime_colours = list("rainbow", "grey", "purple", "metal", "orange",
-	"blue", "dark blue", "dark purple", "yellow", "silver", "pink", "red",
-	"gold", "green", "adamantine", "oil", "light pink", "bluespace",
-	"cerulean", "sepia", "black", "pyrite")
+	var/static/list/slime_colours = list(
+		"rainbow",
+		"grey",
+		"purple",
+		"metal",
+		"orange",
+		"blue",
+		"dark blue",
+		"dark purple",
+		"yellow",
+		"silver",
+		"pink",
+		"red",
+		"gold",
+		"green",
+		"adamantine",
+		"oil",
+		"light pink",
+		"bluespace",
+		"cerulean",
+		"sepia",
+		"black",
+		"pyrite",
```

Comment:
alphabetize this list since you've already gone through the trouble of multilining it

Reply:
You're dead to me

Reply:
i'm right